src/event.rs

```diff
@@ -0,0 +1,258 @@
+use core_foundation::base::{CFRelease, CFRetain, CFTypeID, CFTypeRef, TCFType};
+use geometry::CGPoint;
+use event_source::{CGEventSource,CGEventSourceRef};
+
+use libc;
+use std::mem;
+use std::ptr;
+
+pub type CGKeyCode = libc::uint16_t;
+
+/// Flags for events
+///
+/// [Ref](http://opensource.apple.com/source/IOHIDFamily/IOHIDFamily-700/IOHIDSystem/IOKit/hidsystem/IOLLEvent.h)
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
+pub enum CGEventFlags {
+  // Device-independent modifier key bits.
+  AlphaShift = 0x00010000,
+  Shift = 0x00020000,
+  Control = 0x00040000,
+  Alternate = 0x00080000,
+  Command = 0x00100000,
+
+  // Special key identifiers.
+  Help = 0x00400000,
+  SecondaryFn = 0x00800000,
+
+  // Identifies key events from numeric keypad area on extended keyboards.
+  NumericPad = 0x00200000,
+
+  // Indicates if mouse/pen movement events are not being coalesced
+  NonCoalesced = 0x00000100,
```

Comment:
Oh... It's also from that same file, but it appears they have misspelled 'coalsesced'. The misspelling of coalesced is probably a typo in their definition; should we follow the same typo for our enum? Hmm

IOLLEvent.h
`#define NX_NONCOALSESCEDMASK	0x00000100`